### ISSUE TITLE
Suggestion: rewording the Drupal bullet point on homepage

### ISSUE THREAD

**Issue by laryn** [backdrop-ops/backdropcms.org#421]
Suggestion, if it can be done without getting too wordy: change this line on the b.org homepage to include the fact that while familiar, the UI is improved and improving vs. D7, and not just a familiar continuation of D7:

> 
<img width="695" alt="Backdrop has a built-in upgrade path from Drupal 7. Working with Backdrop will be very familiar to people with Drupal experience." src="https://user-images.githubusercontent.com/871421/29153324-8fd95ba0-7d52-11e7-86d4-436c793664e6.png">

@klonos suggested adding something like this:

>  In fact, because we have improved and continue improving the user interface, chances are that you will find working with Backdrop much easier than Drupal 7.

I like that direction but wonder if an entire new sentence added would make that bullet point seem bloated compared to the others. What about just a new clause at the end like this:

> Backdrop has a built-in upgrade path from Drupal 7. Working with Backdrop will be very familiar to people with Drupal experience, but with an improved interface and new features.

Anybody else have thoughts in general or in particular?

**Comment by klonos** [comment]
Adding @jenlampton and @tomgrandy to this ticket, but anyone else please feel free to make suggestions.

**Comment by jenlampton** [comment]
I bet we can find a way to do it without making it too wordy... How about this?
```
Backdrop has a built-in upgrade path from Drupal 7. Backdrop's improved user-interface and new features will be very familiar to people with Drupal experience.
```

**Comment by laryn** [comment]
I was going for "familiar...but better" -- ie. the improvements and new features may not be familiar. Does that make sense?

**Comment by klonos** [comment]
How about:

>Thanks to the ongoing improvements to the user interface, you will find working with Backdrop to be familiar, and at the same time more intuitive than Drupal 7.

I think that people might also dig a catch phrase like "Drupal 7 on steroids!"

**Comment by jackaponte** [comment]
Hehe...well, let's try to avoid _all_ sorts of negative connotations when coming up with catchphrases, including cultural appropriation and the rather mixed bag of issues when it comes to Gandhi himself. But we digress. :)

I like what @laryn suggested here:

> Backdrop has a built-in upgrade path from Drupal 7. Working with Backdrop will be very familiar to people with Drupal experience, but with an improved interface and new features.

What @klonos suggested is quite similar, but I like the words "more intuitive" to describe the transition from D7 to Backdrop. I often tell folks that Backdrop feels a lot like how I normally configure a D7 site but without all the extra work of adding on contrib modules and configurations that should just be there from the start.

**Comment by jenlampton** [comment]
> Backdrop has a built-in upgrade path from Drupal 7. Working with Backdrop will be very familiar to people with Drupal experience, but with an improved interface and new features.

I have a problem with this grammar... `Working with Backdrop` does not have 'an improved interface...`

**Comment by klonos** [comment]
I don't like repeating the word "Backdrop" in every single sentence of such a small paragraph where we talk about the same thing (adds to the wordyness). How about something like this instead:

> Backdrop has a built-in upgrade path from Drupal 7, and thanks to its similar interface you will feel as if you are simply using a refreshed and modernised admin theme. In fact, we have added many improvements and fixed so many annoyances, that working with Backdrop will be a breeze!

Not fixing the wordyness, but taking advantage of it :)

**Comment by jenlampton** [comment]
I'd prefer not to say anything negative about Drupal. We'd need to exclude `and fixed so many annoyances` at least.

**Comment by klonos** [comment]
But the truth is we have 😞 ...I get what you're saying though. Let me ponder...

**Comment by jenlampton** [comment]
How about we just add `usability` in front of `improvements` in `we have added many improvements `.

**Comment by klonos** [comment]
>Backdrop has a built-in upgrade path from Drupal 7, and thanks to its similar interface you will feel as if you are simply using a refreshed and modernised admin theme. In fact, we have added so many usability improvements, that working with Backdrop will feel like a breeze!

Like so? ^^
